addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -138,6 +138,9 @@
             </t>
         </div>
         <button t-if="event.event_registrations_open" type="button" data-bs-toggle="modal" data-bs-target="#modal_ticket_registration" class="btn btn-primary w-100">Register</button>
+        <div t-if="registration_error_code == 'insufficient_seats'" class="alert alert-danger my-3" role="alert">
+            <span>Registration failed! These tickets are not available anymore.</span>
```

Comment:
For changes that are translatable in stable you need to add `.pot` (translation template) fields manually. It's a bit silly but that's how it is ^^'

You can export the translation here (in debug mode), exporting "website_event"
Then you take the diff relevant to your change and add it in the .pot file at the relevant place (alphabetical order ideally), keeping no absolute red diff (users need to keep translations if they don't update, version doesn't need to change, date doesn't need to change)
![image](https://github.com/odoo/odoo/assets/43653136/a07f6a19-a607-4680-9d2c-e45b6fa61026)